browser-test/test.browser.ts

```diff
@@ -14,8 +14,15 @@
 import assert from 'assert';
 import {describe, it} from 'mocha';
 import {request} from '../src/index';
+import {PassThrough} from 'stream';
```

Comment:
Can we treat lint warnings as errors?

Reply:
I had this chat in the weekly meeting today with @zamnuts, based on your feedback, we're thinking that we should move several of the warning level linting rules in gts to errors, moving towards `3.x.x`. But, as you mentioned, we have a bit of linting fatigue right now, so it would be good to hold off on `3.x.x` for a little while.